slo-monitor/src/monitors/pod_monitor.go

```diff
@@ -130,7 +130,15 @@ func (pm *PodStartupLatencyDataMonitor) Run(stopCh chan struct{}) error {
 		&cache.ListWatch{
 			ListFunc: func(options metav1.ListOptions) (runtime.Object, error) {
 				// This call is paginated by cache.Reflector.
-				return pm.kubeClient.CoreV1().Pods(v1.NamespaceAll).List(options)
+				o := metav1.ListOptions{
+					Limit: 1,
+				}
+				result, err := pm.kubeClient.CoreV1().Pods(v1.NamespaceAll).List(o)
+				if err != nil {
+					return nil, err
+				}
+				result.Continue = ""
```

Comment:
Let's also remove the Items of the list - we don't really list them - it will be easier to catch errors if someone will change other parts.

Reply:
Done.